modules/virt-about-node-maintenance.adoc

```diff
@@ -7,6 +7,11 @@
 
 Nodes can be placed into maintenance mode using the `oc adm` utility, or using `NodeMaintenance` custom resources (CRs).
 
+[NOTE]
+====
+As of version 4.11, `node-maintenance-operator` (NMO) is no longer shipped with {VirtProductName}. It is now available to deploy as a standalone operator through the Operator Lifecycle Manager (OLM) in the OperatorHub.
```

Comment:
I am wondering if it might be better to get rid of the phrase "As of version 4.11" to keep things simple for future docs